session-keys/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+authors = ['Manta Network']
+description = "Primitives for session keys"
+edition = "2021"
+name = "session-key-primitives"
+version = "0.1.0"
+
+[dependencies]
+manta-primitives = { path = "../primitives", default-features = false }
+nimbus-primitives = { git = "https://github.com/manta-network/nimbus.git", branch = "manta-v0.9.18", default-features = false }
+parity-scale-codec = { version = "3.0.0", default-features = false, features = ["derive"] }
+scale-info = { version = "2.0", default-features = false, features = ["derive"] }
+sp-application-crypto = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.18", default-features = false }
+sp-consensus-aura = { git = 'https://github.com/paritytech/substrate.git', default-features = false, branch = "polkadot-v0.9.18" }
+sp-core = { git = 'https://github.com/paritytech/substrate.git', default-features = false, branch = "polkadot-v0.9.18" }
+sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.18", default-features = false }
+
+[features]
```

Comment:
If nimbus will be benchmarked, I suppose you will add `runtime-benchmarks` in the future.

Reply:
yes, once there is something to benchmark